src/niscope/system_tests/test_system_niscope.py

```diff
@@ -294,9 +261,9 @@ def test_waveform_processing(session):
     session.configure_horizontal_timing(50000000, test_record_length, 50.0, test_num_records, True)
     with session.initiate():
         session.add_waveform_processing(niscope.enums.ArrayMeasurement.DERIVATIVE)
-        processed_waveforms = session.channels[test_channels].fetch_measurement(niscope.enums.ScalarMeasurement.MID_REF_VOLTS, 5.0)
+        processed_waveforms = session.channels[test_channels]._fetch_measurement(niscope.enums.ScalarMeasurement.MID_REF_VOLTS, 5.0)
```

Comment:
Not a good idea to use private methods in system tests. You can use `fetch_measurement_stats` instead, right?